composer.json

```diff
@@ -13,6 +13,12 @@
             "name": "Gr4vy"
         }
     ],
+    "repositories": {
+        "composer-magento": {
+            "type": "composer",
+            "url": "https://repo.magento.com/"
+        }
+    },
```

Comment:
@steve-gr4vy why did we have to add this?

Reply:
I had to add this to run composer install in the repo directory.. you're correct that it's not needed for the plugin use-case.. I'll remove it